### DISCUSSION TITLE
Cloudwatch from AWS/AmplifyHosting 4xx and 5xx error codes not showing.

### DISCUSSION THREAD

**Discussion by mwarkyangelo** [grafana/grafana#48740] [Data Sources]
Grafana v8.3.5

The image in the link describes that grafana can display the request metric similar to the cloudwatch on the left-hand side of the image, but the 4xx metric is not being displayed at all. https://i.stack.imgur.com/tTLKx.png
![1](https://user-images.githubusercontent.com/98999511/166877684-f2a62c7a-1718-447a-94ce-04039c54c636.png)

In the 2nd photo, it can be seen that both of the metrics are in the same application. https://i.stack.imgur.com/SbSHi.png
![2](https://user-images.githubusercontent.com/98999511/166877714-36740181-e48a-4e20-87a3-b5a111f09519.png)

Here are some additional details on the metric query https://i.stack.imgur.com/BiKkC.png
![3](https://user-images.githubusercontent.com/98999511/166877725-a8ff610e-9a12-4885-956e-f81920609857.png)

https://i.stack.imgur.com/GXSoQ.png
![4](https://user-images.githubusercontent.com/98999511/166877730-2f976dac-fa0b-4dfe-b14f-170e52ef86df.png)

Here's the error code when I'm using an imported dashboard template: https://grafana.com/grafana/dashboards/590 I tried searching the error code but I'm on my wits end.

https://i.stack.imgur.com/Aki2K.png
![5](https://user-images.githubusercontent.com/98999511/166877744-50345fbf-1ea8-4e3b-9cfd-808fc988f5ed.png)

message:"metric request error: "InvalidParameter: 1 validation error(s) found.\n- minimum field size of 1, GetMetricDataInput.MetricDataQueries[0].MetricStat.Metric.Dimensions[0].Value.\n""

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
